src/scss/base/_base.scss

```diff
@@ -0,0 +1,20 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  font-size: 62.5%;
```

Comment:
czy rem nie bedzie lepsze tutaj?

Reply:
nie 😸  bo to właśnie chodzi o to żeby ustalić domyślny font na 10 a nie na 16 i dzięki temu łatwiej jest się posługiwać rem... bo 12 = 1.2 rem itd. - jestem leniwa i nie chce mi się liczyć rem z 16 🔢